### ISSUE TITLE
Shortcuts: Errors and warnings when deserializing hotkeys from file

### ISSUE THREAD

**Issue by SmithTom6304** [YoYoGames/GameMaker-Bugs#5423]
### Description

**Summary:** Multiple errors and warnings are generated by both `default_hotkeys.json` and `mac_hotkeys.json` when deserializing hotkeys:

```
[16:06:39:796(de1a)] Error deserializing Hotkeys in file './default_hotkeys.json' - Key entry Shift is not a valid Key
[16:06:39:797(de1a)] Error deserializing Hotkeys in file './default_hotkeys.json' - Key entry Control is not a valid Key
[16:06:39:797(de1a)] Error deserializing Hotkeys in file './default_hotkeys.json' - Key entry Shift is not a valid Key
[16:06:39:797(de1a)] Error deserializing Hotkeys in file './default_hotkeys.json' - Key entry Control is not a valid Key
[16:06:39:797(de1a)] Error deserializing Hotkeys in file './default_hotkeys.json' - Key entry Shift is not a valid Key
[16:06:39:798(de1a)] Warning loading hotkeys from file ./default_hotkeys.json - the following hotkeys had multiple entries:
[16:06:39:798(de1a)] 	- 'Select Rotate Tool'
[16:06:39:798(de1a)] 	Duplicate entries will not be loaded
[16:06:39:800(de1a)] Error deserializing Hotkeys in file './mac_hotkeys.json' - Key entry Command is not a valid Key
[16:06:39:800(de1a)] Error deserializing Hotkeys in file './mac_hotkeys.json' - Key entry Command is not a valid Key
[16:06:39:800(de1a)] Error deserializing Hotkeys in file './mac_hotkeys.json' - Key entry Command is not a valid Key
[16:06:39:800(de1a)] Error deserializing Hotkeys in file './mac_hotkeys.json' - Key entry Command is not a valid Key
[16:06:39:800(de1a)] Error deserializing Hotkeys in file './mac_hotkeys.json' - Key entry Command is not a valid Key
[16:06:39:800(de1a)] Error deserializing Hotkeys in file './mac_hotkeys.json' - Key entry Command is not a valid Key
[16:06:39:800(de1a)] Error deserializing Hotkeys in file './mac_hotkeys.json' - Key entry Command is not a valid Key
[16:06:39:800(de1a)] Error deserializing Hotkeys in file './mac_hotkeys.json' - Key entry Command is not a valid Key
[16:06:39:800(de1a)] Error deserializing Hotkeys in file './mac_hotkeys.json' - Key entry Command is not a valid Key
[16:06:39:800(de1a)] Error deserializing Hotkeys in file './mac_hotkeys.json' - Key entry Alt is not a valid Key
[16:06:39:800(de1a)] Error deserializing Hotkeys in file './mac_hotkeys.json' - Key entry Command is not a valid Key
[16:06:39:800(de1a)] Error deserializing Hotkeys in file './mac_hotkeys.json' - Key entry Command is not a valid Key
[16:06:39:801(de1a)] Warning loading hotkeys from file ./mac_hotkeys.json - the following hotkeys had multiple entries:
[16:06:39:801(de1a)] 	- 'Home'
[16:06:39:801(de1a)] 	- 'End'
[16:06:39:801(de1a)] 	- 'Home + Select'
[16:06:39:801(de1a)] 	- 'End + Select'
[16:06:39:801(de1a)] 	Duplicate entries will not be loaded
```

The errors here were introduced from beta `IDE v2024.400.0.533`, following [the recent changes to Shortcuts](https://github.com/YoYoGames/YoYoStudio/pull/9812). The warnings have been around for longer. We should resolve all of these.

[ui.log](https://github.com/YoYoGames/GameMaker-Bugs/files/14934108/ui.log)

### Expected Change

No errors or warnings relating to hotkeys should be found in the ui.log on startup

### Steps To Reproduce

1. Start GameMaker
2. View the `ui.log`
3. See the issue

### How reliably can you recreate this issue using your steps above?

Always

### Which version of GameMaker are you reporting this issue for?

2024.400 (Betas)

### Which operating system(s) are you seeing the problem on?

macOS Ventura

### Are you running GameMaker from inside your Steam library?

None

### Contact Us Package Attached?

- [X] I have attached my Contact Us Package

### Sample Project Added?

- [ ] I have included a small sample project

**Comment by SmithTom6304** [comment]
All errors and warnings should now be fixed. Duplicate entries for "Select Rotate Tool" have now been split as follows:
> Select Rotate Image Tool
Used in `Image Editor` and `Tile Set Editor`
Bound to `Z`

> Select Rotate Sequence Tool
Used in `Sequence Editor`
Bound to `R`

**Comment by SamTHodson** [comment]
Verified fixed in IDE v2024.400.0.539  Runtime v2024.400.0.558
